packages/embla-carousel/src/CarouselProgressLabel.tsx

```diff
@@ -32,31 +32,31 @@ export function CarouselProgressLabel({
   const [currentSlide, setCurrentSlide] = useState("");
   const [totalSlides, setTotalSlides] = useState(0);
 
-  const handleSelect = useCallback((emblaApi: EmblaCarouselType) => {
-    const slideIndexInView = emblaApi?.selectedScrollSnap() ?? 0;
-    const numberOfSlides = emblaApi?.slideNodes().length ?? 0;
-    const scrollSnaps = emblaApi?.scrollSnapList() ?? [];
-    const slidesPerTransition = numberOfSlides
-      ? Math.ceil(numberOfSlides / scrollSnaps.length)
-      : 0;
-    const startSlideNumber = Math.min(
-      slideIndexInView * slidesPerTransition + 1,
-      numberOfSlides - (slidesPerTransition - 1),
-    );
-    const endSlideNumber = Math.min(
-      startSlideNumber + slidesPerTransition - 1,
-      numberOfSlides,
-    );
+  useEffect(() => {
+    const handleSelect = (emblaApi: EmblaCarouselType) => {
```

Comment:
Moving handleSelect in removes the need for useCallback

Reply:
nice